docker-compose.yml

```diff
@@ -11,7 +11,7 @@ services:
     image: 'redis:alpine'
     ports:
       - '6379:6379'
-    command: ['redis-server', '--bind', 'redis', '--port', '$REDIS_PORT']
+    command: ['redis-server', '--bind', 'redis', '--port', '6379']
```

Comment:
same for this right. We shouldn't need to replace the ENVIRONMENT VARIABLE

Reply:
We are already telling that we are using 6379 in the above(13th) line, then why should we add the env value in 14th line. I didn't understand that.

Reply:
we should change the outbound port based on the env variable on line 13 as well